pkg/kubelet/kubelet_test.go

```diff
@@ -874,11 +874,6 @@ func TestSyncPodDeletesDuplicate(t *testing.T) {
 			Names: []string{"/k8s_foo_bar.new.test_12345678_3333"},
 			ID:    "4567",
 		},
-		"2304": &docker.APIContainers{
```

Comment:
Aaaaah right because this would no longer exist now. Got it. Would be nice if we have this test elsewhere, but that may be too hard for now.